engine/apps/webhooks/tasks/trigger_webhook.py

```diff
@@ -122,6 +122,13 @@ def _build_payload(
 
     data = serialize_event(event, alert_group, user, webhook, responses_data)
 
+    # TODO: find a better way to filter, additional_settings__isnull is a workaround
+    source_alert_receive_channel = webhook.filtered_integrations.filter(additional_settings__isnull=False).first()
+    if source_alert_receive_channel and hasattr(source_alert_receive_channel.config, "additional_template_context"):
+        data.update(
+            source_alert_receive_channel.config.additional_template_context(alert_group, source_alert_receive_channel)
+        )
```

Comment:
Would it make sense to update the `serialize_event` helper to add the external_ids (and the corresponding source integration settings) in the context of the alert group, using the external ID model? (ie. does it make sense to do [this](https://github.com/grafana/oncall-private/pull/2581/files#diff-3a474aa4e2ce1b1a0b2c6aaa9483665b3c3eb5b0e5ea10349a743d74bbfcb9d4R200) for all external IDs related to the alert group?)

Also, it will be nice to add some test for this.